serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -43,7 +47,18 @@ minikube start [...] --driver podman
 
 == Install the {product_name} Operator
 
-In order to have an up-and-running instance of the {product_name} Operator you can use the following command:
+In order to have an up-and-running instance of the {product_name} Operator you can use the following commands:
+
+.Install Cert Manager Operator
+[source,shell,subs="attributes+"]
+----
+kubectl apply -f {cert_manager_installer_url}
+----
+[NOTE]
+====
+The Cert Manager is optional, if you choose not to install it, you need to make sure that the certificates needed by the Webhooks are in place, more information can be found link:{kubebuilder_webhooks_url}[here].
```

Comment:
If it's optional, we should provide an alternative way for users to "make sure that the certificates needed by the Webhooks are in place". Hence, some instructions.